client/cli/src/lib.rs

```diff
@@ -47,13 +45,11 @@ use structopt::{
 	clap::{self, AppSettings},
 	StructOpt,
 };
-use tracing_subscriber::{
-	filter::Directive, fmt::time::ChronoLocal, layer::SubscriberExt, FmtSubscriber, Layer,
-};
-
-pub use logging::PREFIX_LOG_SPAN;
+pub use sc_logging::{PREFIX_LOG_SPAN, prefix_logs_with};
 #[doc(hidden)]
 pub use tracing;
+#[cfg(test)]
+use trybuild as _;
```

Comment:
Why do we need this?

Reply:
> Probably don't need it, so this can be removed.

To answer to your 2 comments:

I get this error if I don't do this:

```
warning: external crate `trybuild` unused in `sc_cli`: remove the dependency or add `use trybuild as _;`
  --> client/cli/src/lib.rs:19:1
   |
19 | //! Substrate CLI library.
   | ^
   |
note: the lint level is defined here
  --> client/cli/src/lib.rs:24:9
   |
24 | #![warn(unused_crate_dependencies)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^
```

I added a trybuild test because I wanted to re-export the macro `prefix_with_logs!` in `sc_cli` so people don't need to explicitly import sc-logging.

Though I'm not sure if it's a good idea. Let me know what you think.

`prefix_with_logs!` was in sc-cli. But now it makes more sense to have it on sc-logging (so it can be use with something that don't use sc-cli).

Reply:
If this weird lint is complaining, remove it!

And yeah, I'm all in for now re-exporting the macro in `sc-cli`.

Reply:
Ok... I'm a bit reluctant to remove it because it's kinda useful. When I was working on this PR it detected a lot of unused dependencies. But ok I'll do it

Reply:
AFAIK it produces a lot of false positives like this one and I'm clearly not for accepting code changes to make a non working lint happy.